backend/product-service/config/config.development.yaml

```
@@ -68,4 +68,9 @@ database:
 
 redis:
   host: "localhost"  # Changed from "redis" to "localhost"
-  port: "6379"
\ No newline at end of file
+  port: "6379"
+
+services:
+  inventory:
+    host: "localhost"
+    port: "50055"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)